### ISSUE TITLE
If I try to use a "poshgit" segment, debug reports poshgit(false)

### ISSUE THREAD

**Issue by pfmoore** [JanDeDobbeleer/oh-my-posh#2355]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

I altered my config to use the poshgit segment in place of git. I have Posh-Git installed and imported in my profile before oh-my-posh. My config has a copy of the default poshgit settings:

```
        {
          "type": "poshgit",
          "style": "powerline",
          "powerline_symbol": "\uE0B0",
          "foreground": "#ffffff",
          "background": "#0077c2"
        },
```

If I run `oh-my-posh debug` when in a git checkout, the output shows:

```
Version: 7.94.0

Segments:

ConsoleTitle(true) -   0 ms - oh-my-posh in pkg_metadata
session(true)  -   0 ms -  Gustav 
path(true)     -   0 ms -   pkg_metadata
poshgit(false) -   0 ms -
root(false)    -   0 ms -
exit(true)     -   0 ms -   
node(false)    -   0 ms -
go(false)      -   0 ms -
python(true)   -  39 ms - 
shell(true)    -   0 ms - in oh-my-posh
time(true)     -   0 ms - at 12:16:53

Run duration: 55.5858ms
```

I can't find any instructions on how to 

### Theme

A custom config. To ensure I didn't have an issue with something else in my config, I replaced it with a copy of `jandedobbeleer.omp.json` and just replaced the git section in that with the poshgit section shown above.

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
[1mVersion:[0m 7.94.0

[1mSegments:[0m

ConsoleTitle(true)   -   0 ms - oh-my-posh in pkg_metadata
session(true)        -   0 ms - [38;2;195;134;241m[0m[48;2;195;134;241m[38;2;255;255;255m Gustav [0m[38;2;195;134;241m[0m
path(true)           -   1 ms - [38;2;255;71;156;49m[7m[0m[48;2;255;71;156m[38;2;255;255;255m   pkg_metadata [0m
poshgit(false)       -   0 ms - 
node(false)          -   0 ms - 
go(false)            -   0 ms - 
julia(false)         -   0 ms - 
python(true)         -  40 ms - [48;2;255;222;87m[38;2;255;71;156m[0m[48;2;255;222;87m[38;2;17;17;17m   [0m
ruby(false)          -   0 ms - 
azfunc(false)        -   0 ms - 
aws(false)           -   0 ms - 
root(false)          -   0 ms - 
executiontime(true)  -   0 ms - [38;2;255;222;87m[0m[38;2;131;118;156;49m[7m[0m[48;2;131;118;156m[38;2;255;255;255m ﮫ0ms⠀[0m
exit(true)           -   1 ms - [48;2;0;137;123m[38;2;131;118;156m[0m[48;2;0;137;123m[38;2;255;255;255m  [0m[38;2;0;137;123m[0m
shell(true)          -   0 ms - [38;2;0;119;194m[0m[48;2;0;119;194m[38;2;255;255;255m  oh-my-posh [0m[38;2;0;119;194;49m[7m[0m
ytm(false)           -  34 ms - 
battery(true)        -   8 ms - [38;2;76;175;80m[0m[48;2;76;175;80m[38;2;255;255;255m  100 [0m
time(true)           -   0 ms - [38;2;76;175;80;49m[7m[0m[38;2;46;149;153m[0m[48;2;46;149;153m[38;2;17;17;17m 12:25:42 [0m[38;2;46;149;153m[0m

[1mRun duration:[0m 97.3423ms

[1mCache path:[0m C:\Users\Gustav\AppData\Local\oh-my-posh

[1mConfig path:[0m C:\Users\Gustav\AppData\Local\oh-my-posh\config.omp.json

[1mLogs:[0m

2022/05/31 12:25:42 debug: Getenv
C:\Users\Gustav\AppData\Local
2022/05/31 12:25:42 Getenv duration: 0s, args: LOCALAPPDATA
2022/05/31 12:25:42 CachePath duration: 636.9µs, args: 
2022/05/31 12:25:42 debug: Getenv
C:\Users\Gustav\AppData\Local\oh-my-posh\config.omp.json
2022/05/31 12:25:42 Getenv duration: 0s, args: POSH_THEME
2022/05/31 12:25:42 Shell duration: 9.3727ms, args: 
2022/05/31 12:25:42 resolveConfigPath duration: 9.3727ms, args: 
2022/05/31 12:25:42 Init duration: 10.0096ms, args: 
2022/05/31 12:25:42 Flags duration: 0s, args: 
2022/05/31 12:25:42 config.loadConfig duration: 985.8µs, args: 
2022/05/31 12:25:42 Flags duration: 0s, args: 
2022/05/31 12:25:42 debug: Getenv

2022/05/31 12:25:42 Getenv duration: 0s, args: OMP_CACHE_DISABLED
2022/05/31 12:25:42 WindowsRegistryKeyValue duration: 0s, args: HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor
2022/05/31 12:25:42 debug: WindowsRegistryKeyValue
WindowsRegistryKeyValue: root:"HKEY_CURRENT_USER", path:"Software\Microsoft\Windows\DWM", key:"ColorizationColor"
2022/05/31 12:25:42 debug: WindowsRegistryKeyValue
success, DWORD, 0xC40078D4
2022/05/31 12:25:42 Root duration: 0s, args: 
2022/05/31 12:25:42 Shell duration: 0s, args: 
2022/05/31 12:25:42 ErrorCode duration: 0s, args: 
2022/05/31 12:25:42 IsWsl duration: 0s, args: 
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 debug: User
Gustav
2022/05/31 12:25:42 User duration: 0s, args: 
2022/05/31 12:25:42 debug: Host
Teemo
2022/05/31 12:25:42 Host duration: 0s, args: 
2022/05/31 12:25:42 GOOS duration: 0s, args: 
2022/05/31 12:25:42 TemplateCache duration: 0s, args: 
2022/05/31 12:25:42 Flags duration: 0s, args: 
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 GOOS duration: 0s, args: 
2022/05/31 12:25:42 debug: Getenv

2022/05/31 12:25:42 Getenv duration: 0s, args: SSH_CONNECTION
2022/05/31 12:25:42 debug: Getenv

2022/05/31 12:25:42 Getenv duration: 0s, args: SSH_CLIENT
2022/05/31 12:25:42 TemplateCache duration: 0s, args: 
2022/05/31 12:25:42 TemplateCache duration: 0s, args: 
2022/05/31 12:25:42 Shell duration: 0s, args: 
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 GOOS duration: 0s, args: 
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 Flags duration: 0s, args: 
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 GOOS duration: 0s, args: 
2022/05/31 12:25:42 GOOS duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 GOOS duration: 0s, args: 
2022/05/31 12:25:42 IsWsl duration: 0s, args: 
2022/05/31 12:25:42 StackCount duration: 0s, args: 
2022/05/31 12:25:42 DirIsWritable duration: 0s, args: 
2022/05/31 12:25:42 TemplateCache duration: 0s, args: 
2022/05/31 12:25:42 TemplateCache duration: 0s, args: 
2022/05/31 12:25:42 Shell duration: 0s, args: 
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 GOOS duration: 0s, args: 
2022/05/31 12:25:42 debug: Getenv

2022/05/31 12:25:42 Getenv duration: 0s, args: POSH_GIT_STATUS
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 GOOS duration: 0s, args: 
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 debug: HasFiles
false
2022/05/31 12:25:42 HasFiles duration: 0s, args: *.js
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 debug: HasFiles
false
2022/05/31 12:25:42 HasFiles duration: 0s, args: *.ts
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 debug: HasFiles
false
2022/05/31 12:25:42 HasFiles duration: 0s, args: package.json
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 debug: HasFiles
false
2022/05/31 12:25:42 HasFiles duration: 0s, args: .nvmrc
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 debug: HasFiles
false
2022/05/31 12:25:42 HasFiles duration: 0s, args: pnpm-workspace.yaml
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 debug: HasFiles
false
2022/05/31 12:25:42 HasFiles duration: 0s, args: .pnpmfile.cjs
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 debug: HasFiles
false
2022/05/31 12:25:42 HasFiles duration: 0s, args: .npmrc
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 debug: HasFiles
false
2022/05/31 12:25:42 HasFiles duration: 0s, args: .vue
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 GOOS duration: 0s, args: 
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 debug: HasFiles
false
2022/05/31 12:25:42 HasFiles duration: 0s, args: *.go
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 debug: HasFiles
false
2022/05/31 12:25:42 HasFiles duration: 0s, args: go.mod
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 GOOS duration: 0s, args: 
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 debug: HasFiles
false
2022/05/31 12:25:42 HasFiles duration: 0s, args: *.jl
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 GOOS duration: 0s, args: 
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 debug: HasFiles
true
2022/05/31 12:25:42 HasFiles duration: 0s, args: *.py
2022/05/31 12:25:42 debug: Getenv
C:\Users\Gustav\AppData\Local
2022/05/31 12:25:42 Getenv duration: 0s, args: LOCALAPPDATA
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 debug: HasFilesInDir
false
2022/05/31 12:25:42 HasFilesInDir duration: 0s, args: python.exe
2022/05/31 12:25:42 error: CommandPath
no Windows Store App
2022/05/31 12:25:42 HasCommand duration: 10.1144ms, args: python
2022/05/31 12:25:42 debug: Getenv
C:\Users\Gustav\AppData\Local
2022/05/31 12:25:42 Getenv duration: 0s, args: LOCALAPPDATA
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 debug: HasFilesInDir
false
2022/05/31 12:25:42 HasFilesInDir duration: 0s, args: python3.exe
2022/05/31 12:25:42 error: CommandPath
no Windows Store App
2022/05/31 12:25:42 HasCommand duration: 8.8417ms, args: python3
2022/05/31 12:25:42 debug: Getenv
C:\Users\Gustav\AppData\Local
2022/05/31 12:25:42 Getenv duration: 0s, args: LOCALAPPDATA
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 debug: HasFilesInDir
false
2022/05/31 12:25:42 HasFilesInDir duration: 0s, args: python.exe
2022/05/31 12:25:42 error: CommandPath
no Windows Store App
2022/05/31 12:25:42 HasCommand duration: 8.9019ms, args: python
2022/05/31 12:25:42 debug: Getenv
C:\Users\Gustav\AppData\Local
2022/05/31 12:25:42 Getenv duration: 0s, args: LOCALAPPDATA
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 debug: HasFilesInDir
false
2022/05/31 12:25:42 HasFilesInDir duration: 0s, args: python3.exe
2022/05/31 12:25:42 error: CommandPath
no Windows Store App
2022/05/31 12:25:42 HasCommand duration: 9.4759ms, args: python3
2022/05/31 12:25:42 TemplateCache duration: 0s, args: 
2022/05/31 12:25:42 TemplateCache duration: 0s, args: 
2022/05/31 12:25:42 Shell duration: 0s, args: 
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 GOOS duration: 0s, args: 
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 debug: HasFiles
false
2022/05/31 12:25:42 HasFiles duration: 0s, args: *.rb
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 debug: HasFiles
false
2022/05/31 12:25:42 HasFiles duration: 0s, args: Rakefile
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 debug: HasFiles
false
2022/05/31 12:25:42 HasFiles duration: 0s, args: Gemfile
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 GOOS duration: 0s, args: 
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 debug: HasFiles
false
2022/05/31 12:25:42 HasFiles duration: 0s, args: host.json
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 debug: HasFiles
false
2022/05/31 12:25:42 HasFiles duration: 0s, args: local.settings.json
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 PathSeparator duration: 0s, args: 
2022/05/31 12:25:42 debug: HasFiles
false
2022/05/31 12:25:42 HasFiles duration: 0s, args: function.json
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 GOOS duration: 0s, args: 
2022/05/31 12:25:42 debug: Getenv

2022/05/31 12:25:42 Getenv duration: 0s, args: AWS_VAULT
2022/05/31 12:25:42 debug: Getenv

2022/05/31 12:25:42 Getenv duration: 0s, args: AWS_PROFILE
2022/05/31 12:25:42 debug: Getenv

2022/05/31 12:25:42 Getenv duration: 0s, args: AWS_REGION
2022/05/31 12:25:42 debug: Getenv

2022/05/31 12:25:42 Getenv duration: 0s, args: AWS_DEFAULT_REGION
2022/05/31 12:25:42 debug: Getenv

2022/05/31 12:25:42 Getenv duration: 0s, args: AWS_CONFIG_FILE
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 error: FileContent
open C:\Users\Gustav/.aws/config: The system cannot find the file specified.
2022/05/31 12:25:42 FileContent duration: 0s, args: C:\Users\Gustav/.aws/config
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 GOOS duration: 0s, args: 
2022/05/31 12:25:42 Root duration: 0s, args: 
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 GOOS duration: 0s, args: 
2022/05/31 12:25:42 ExecutionTime duration: 0s, args: 
2022/05/31 12:25:42 TemplateCache duration: 0s, args: 
2022/05/31 12:25:42 TemplateCache duration: 0s, args: 
2022/05/31 12:25:42 Shell duration: 0s, args: 
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 GOOS duration: 0s, args: 
2022/05/31 12:25:42 ErrorCode duration: 0s, args: 
2022/05/31 12:25:42 TemplateCache duration: 0s, args: 
2022/05/31 12:25:42 TemplateCache duration: 0s, args: 
2022/05/31 12:25:42 Shell duration: 0s, args: 
2022/05/31 12:25:42 TemplateCache duration: 0s, args: 
2022/05/31 12:25:42 Flags duration: 0s, args: 
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 GOOS duration: 0s, args: 
2022/05/31 12:25:42 Shell duration: 0s, args: 
2022/05/31 12:25:42 Flags duration: 0s, args: 
2022/05/31 12:25:42 TemplateCache duration: 0s, args: 
2022/05/31 12:25:42 TemplateCache duration: 0s, args: 
2022/05/31 12:25:42 Shell duration: 0s, args: 
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 GOOS duration: 0s, args: 
2022/05/31 12:25:42 error: HTTPRequest
Get "http://127.0.0.1:9863/query": context deadline exceeded
2022/05/31 12:25:42 HTTPRequest duration: 34.7194ms, args: http://127.0.0.1:9863/query
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 GOOS duration: 0s, args: 
2022/05/31 12:25:42 IsWsl duration: 0s, args: 
2022/05/31 12:25:42 BatteryInfo duration: 7.8801ms, args: 
2022/05/31 12:25:42 TemplateCache duration: 0s, args: 
2022/05/31 12:25:42 TemplateCache duration: 0s, args: 
2022/05/31 12:25:42 Shell duration: 0s, args: 
2022/05/31 12:25:42 TemplateCache duration: 0s, args: 
2022/05/31 12:25:42 TemplateCache duration: 0s, args: 
2022/05/31 12:25:42 TemplateCache duration: 0s, args: 
2022/05/31 12:25:42 debug: Pwd
C:\Work\Projects\pkg_metadata
2022/05/31 12:25:42 Pwd duration: 0s, args: 
2022/05/31 12:25:42 debug: Home
C:\Users\Gustav
2022/05/31 12:25:42 GOOS duration: 0s, args: 
2022/05/31 12:25:42 TemplateCache duration: 0s, args: 
2022/05/31 12:25:42 TemplateCache duration: 0s, args: 
2022/05/31 12:25:42 Shell duration: 0s, args: 
2022/05/31 12:25:42 debug: Getenv
C:\Users\Gustav\AppData\Local
2022/05/31 12:25:42 Getenv duration: 0s, args: LOCALAPPDATA
2022/05/31 12:25:42 CachePath duration: 0s, args: 
2022/05/31 12:25:42 Flags duration: 0s, args:
```

**Comment by JanDeDobbeleer** [comment]
@pfmoore there seems to be something lacking in the docs. You need to set the following environment variable: `$env:POSH_GIT_ENABLED = $true`

**Comment by pfmoore** [comment]
Ah cool, thanks. I did try to check the docs and didn't see that. I found it now, it's under the docs for the "git" (not "poshgit") segment.

**Comment by JanDeDobbeleer** [comment]
Reopening as I will fix that!

**Comment by pfmoore** [comment]
Actually, even now that I've set `$env:POSH_GIT_ENABLED` it seems to behave weirdly - it always puts the Posh-Git status at the start of the prompt line, instead of where I position it in the template. Is that to be expected?

Also, `oh-my-posh debug` still shows it as `poshgit(false)` all the time - I guess that may be because it needs fancy integration with the Powershell environment that debug doesn't have, but of so, how do I check whether the performance benefit of using it is worth it over just using the git segment?

I will say that I've just discovered that if I leave `fetch_status` at its default of `false`, the `git` segment is actually plenty fast enough, so I'm probably not going to bother with `poshgit` anyway. As a result this is mostly just theoretical to me at this point, so don't spend too much time on it!
